engine/src/main/java/org/destinationsol/game/StarPort.java

```diff
@@ -73,7 +73,7 @@ private static void blip(SolGame game, SolShip ship) {
     @Bound
     public static Vector2 getDesiredPosition(Planet from, Planet to, boolean precise) {
         Vector2 fromPosition = from.getPosition();
-        float angle = SolMath.angle(fromPosition, to.getPosition(), precise);
+        float angle = SolMath.angle(fromPosition, to.getPosition());
```

Comment:
Why did we convert all non-precise calls to precise ones?

Reply:
There were two cases of precise/non-precise functions. In one case, the precise executed _exactly_ the same code, in the other, it was told in javadoc a several layers upper, that the precise version is quite slower, while the non-precise one has error of on average 0.01 degrees.